spec/21-http_response_header_format.md

```diff
@@ -0,0 +1,184 @@
+
+Vendors MUST expect the header name in any case (upper, lower, mixed), and SHOULD send the header name in lowercase.
+
+### tracemeta Header Field Values
+
+
+This section uses the Augmented Backus-Naur Form (ABNF) notation of [[!RFC5234]], including the DIGIT rule from that document. The `DIGIT` rule defines a single number character `0`-`9`.
+
+``` abnf
+HEXDIGLC = DIGIT / "a" / "b" / "c" / "d" / "e" / "f" ; lowercase hex character
+value           = version "-" version-format
+```
+
+The dash (`-`) character is used as a delimiter between fields.
+
+#### version
+
+``` abnf
+version         = 2HEXDIGLC   ; this document assumes version 00. Version 255 is forbidden
+```
+
+The value is US-ASCII encoded (which is UTF-8 compliant).
+
+Version (`version`) is 1 byte representing an 8-bit unsigned integer. Version `255` is invalid. The current specification assumes the `version` is set to `00`.
+
+#### version-format
+
+The following `version-format` definition is used for version `00`.
+
+``` abnf
+version-format   = [trace-id] "-" [parent-id] "-" [trace-flags]
+trace-id         = 32HEXDIGLC  ; 16 bytes array identifier. All zeroes forbidden
+parent-id        = 16HEXDIGLC  ; 8 bytes array identifier. All zeroes forbidden
+trace-flags      = 2HEXDIGLC   ; 8 bit flags. Currently, only one bit is used. See below for details
+```
+
+#### trace-id
+
+This is the ID of the whole trace forest and is used to uniquely identify a <a href="#dfn-distributed-traces">distributed trace</a> through a system. It is represented as a 16-byte array, for example, `4bf92f3577b34da6a3ce929d0e0e4736`. All bytes as zero (`00000000000000000000000000000000`) is considered an invalid value.
+
+If the `trace-id` value is invalid (for example if it contains non-allowed characters or all zeros), vendors MUST ignore the `tracemeta`.
+
+See [considerations for trace-id field
+generation](#considerations-for-trace-id-field-generation) for recommendations
+on how to operate with `trace-id`.
+
+#### parent-id
+
+This is the ID of the calling request as known by the callee (in some tracing systems, this is known as the `span-id`, where a `span` is the execution of a client request). It is represented as an 8-byte array, for example, `00f067aa0ba902b7`. All bytes as zero (`0000000000000000`) is considered an invalid value.
+
+Vendors MUST ignore the `tracemeta` when the `parent-id` is invalid (for example, if it contains non-lowercase hex characters).
+
+#### trace-flags
+
+An <a data-cite='!BIT-FIELD#firstHeading'>8-bit field</a>  that controls tracing flags such as sampling, trace level, etc. These flags are recommendations given by the callee rather than strict rules to follow for three reasons:
+
+1. Trust and abuse
+2. Bug in the callee
```

Comment:
This is not a sentence

Reply:
Taken straight from https://github.com/dynatrace-oss-contrib/trace-context/blob/add-response-headers/spec/20-http_request_header_format.md#trace-flags

If we update it here we should update it there too WDYT

Reply:
@yurishkuro do you think we should update this in both places (request and response headers), only update it here, or is it fine to just leave?